acceleration/TensorRT_inference_acceleration.ipynb

```diff
@@ -0,0 +1,930 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%cd spleen_ct_segmentation\n",
+    "!python -m monai.bundle trt_export network_def --filepath models/model_trt_fp32.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json --precision fp32 --dynamic_batchsize \"[1, 4, 8]\"\n",
+    "!python -m monai.bundle trt_export network_def --filepath models/model_trt_fp16.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.json --precision fp16 --dynamic_batchsize \"[1, 4, 8]\"\n",
+    "%cd .."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load the torch model and TensorRT model\n",
+    "\n",
+    "Load the original torch model and the TensorRT engine-based TorchScript models. Please note that `torch_tensorrt>=1.4.0` must be imported before loading TensorRT engine-based TorchScript models, since it will provide some runtime context."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def get_parser(meta_config, inference_config):\n",
+    "    parser = monai.bundle.ConfigParser()\n",
+    "    parser.read_meta(meta_config)\n",
+    "    parser.read_config(inference_config)\n",
+    "    return parser\n",
+    "\n",
+    "\n",
+    "device = torch.device(\"cuda\" if torch.cuda.is_available() else \"cpu\")\n",
+    "model_weight = glob.glob(os.path.join(\"./spleen_ct_segmentation\", \"*\", \"model.pt\"))[0]\n",
+    "meta_config = glob.glob(os.path.join(\"./spleen_ct_segmentation\", \"*\", \"metadata.*\"))[0]\n",
+    "inference_config = glob.glob(os.path.join(\"./spleen_ct_segmentation\", \"*\", \"inference.*\"))[0]\n",
+    "trt_fp32_model_path = os.path.join(\"./spleen_ct_segmentation\", \"models\", \"model_trt_fp32.ts\")\n",
+    "trt_fp16_model_path = os.path.join(\"./spleen_ct_segmentation\", \"models\", \"model_trt_fp16.ts\")\n",
+    "\n",
+    "parser = get_parser(meta_config, inference_config)\n",
+    "\n",
+    "# If using other bundles, the net_id and spatial_shape may change. Please modify these variables accordingly.\n",
+    "net_id = \"network\"\n",
+    "spatial_shape = (1, 1, 96, 96, 96)\n",
+    "model = parser.get_parsed_content(net_id)\n",
+    "\n",
+    "model.eval()\n",
+    "model.to(device)\n",
```

Comment:
No need to move, the parsed `network` object of bundle is already on the target device.

Thanks.